docs/tutorial.rst

```diff
@@ -332,7 +332,7 @@ incompatible versions of packages already installed with conda.
 
 .. code-block:: python
 
-    session.install("contexter", "--no-deps")
+    session.install("contextlib", "--no-deps")
```

Comment:
I'd suggest to leave this unchanged. This is an example in the documentation for an arbitrary third-party library.

Also this does not work, contextlib cannot be installed from PyPI. Nor does it need to be, it's part of the standard library.